rmf_traffic/src/rmf_traffic/schedule/Patch.cpp

```diff
@@ -81,7 +81,9 @@ class Patch::Implementation
 
   std::vector<Change::UnregisterParticipant> unregistered;
   std::vector<Change::RegisterParticipant> registered;
+  std::vector<Change::UpdateParticipantInfo> updated;
```

Comment:
We should be able to get rid of all these fields related to participants since we don't need participant info in the patch anymore.